app/controllers/statistics_announcements_controller.rb

```diff
@@ -31,12 +21,4 @@ def expire_cache_for_index_on_next_announcement_expiry(announcements)
     time_to_releases = announcements.map {|ann| ann.release_date - Time.zone.now }.reject {|time_span| time_span <= 0 }
     expires_in((time_to_releases << Whitehall.default_cache_max_age).min)
   end
-
-  def expire_cache_for_show_on_release(announcement)
```

Comment:
I'm not sure the new `publishing-api` version of Statistics Announcements gives us quite the same behaviour as `expire_cache_for_show_on_release`. The behaviour tries to minimise the time between a Statistical Announcement's document being published, and the announcement URL redirecting to the newly published Statistics document.

I chatted to @fofr who explained Whitehall with publish a `redirect` for the announcement `base_path`, which will be picked up by the router, which ignores the rendering application and makes sense in the new world :+1: 

But, the behaviour here is to start returning a shorter cache expiry time as the release date approaches. It will issue a cache expiry time that is the shortest of;

- the planned/exact release date on the announcement
- the scheduled release date of the announcements child publication (if it's scheduled)
- the default whitehall expiry (currently, 15 minutes)

So that as a planned/scheduled release date approaches the default expiry time is reduced, eg if this page is fetched from origin _3 minutes_ before the planned/scheduled release date, then the cache expiry will be set to _3 minutes_, ensuring the announcement URL will be refetched from origin as close to the release date as possible.

With the new implementation the statistics announcement URL will always be cached with the default expiry time, which may cause the URL to be cached for up to 15 minutes after the planned/scheduled release date, which would be a regression.